### PR TITLE
Update references

### PR DESCRIPTION
**Pull request by StefanWerW**
choonkending/react-webpack-node --> reactGo/reactGo

Should also update the name in app.json and package.json ?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "react-webpack-node",
```
Comment (StefanWerW) [inline package.json]: Maybe change to ReactGo?

File: app.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "React Node Express Mongo",
```
Comment (StefanWerW) [inline app.json]: Change to ReactGo?

File: package.json (hunk 1 above)
Comment (ZeroCho) [inline package.json]: Good catch!
@choonkending

File: app.json (hunk 2 above)
Comment (choonkending) [inline app.json]: Sounds right 💃

**Comment by StefanWerW** [comment]
@choonkending The name in `package.json` and `app.json`, should I update it, or you'll do it? 😄

**Comment by choonkending** [comment]
It's a race! I probably wont get to it in time cause I'm prepping some slides :P

**Pull request merged**